auth/auth-validation.js

```diff
@@ -0,0 +1,151 @@
+import { ERROR_MESSAGES } from '../constants/auth-validation-messages.js';
+
+document.addEventListener('DOMContentLoaded', function () {
+  const form = document.querySelector('.auth-form');
+  const authSubmitButton = document.querySelector('.auth-button');
+  const emailInput = document.getElementById('email');
+  const passwordInput = document.getElementById('password');
+  const nicknameInput = document.getElementById('nickname');
+  const confirmPasswordInput = document.getElementById('password-confirm');
+  const authType = form.dataset.authType;
+
+  let emailValue;
+  let nicknameValue;
+  let passwordValue;
+  let confirmPasswordValue;
+
+  // 이메일 형식 검증
+  const validateEmail = (email) => {
+    const regex = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
+    return regex.test(String(email));
+  };
+
+  // 에러 메시지 표시
+  const toggleError = (targetInput, message, isInputValid) => {
+    const inputContainer = targetInput.closest('.input-container');
+    if (!inputContainer) return;
+    const errorContainer = inputContainer.querySelector(
+      '.validation-error-message',
+    );
```

Comment:
💊 제안
inputContainer 변수가 한번만 사용되고 있어 변수로 저장할 필요성이 낮아보여요~
아래처럼 작성하시면 더 가독성측면에서 좋을 것 같아요~
변수는 최소 2번 이상 사용하거나 변수로 선언했을 때 가독성에서 유리할 것 같은 경우만 선언하시는 것을 추천드려요!

```suggestion
    const errorContainer = targetInput.closest('.input-container')?.querySelector('.validation-error-message');
    if (!errorContainer) return;
```